src/redis_mod.f90

```diff
@@ -2,6 +2,13 @@ module redis_mod
   use iso_c_binding
   implicit none
 
+! Test for a little-endian machine
```

Comment:
Is the ` __BYTE_ORDER__ ` macro portable across different fortran compilers? Also, if we use macros, we should change the file extension from `.f90` to `.F90`.